spejstore/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+services:
+  db:
+    image: ghcr.io/hackerspacekrk/spejstore/postgres-hstore:latest
+    build: postgres-hstore
+    restart: always
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - spejstore-pg:/var/lib/postgresql/data
+  web:
+    image: ghcr.io/hackerspacekrk/spejstore:latest
+    restart: always
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
+    secrets:
+      - spejstore_clientid
+      - spejstore_secret
+    environment:
+      SPEJSTORE_CLIENT_ID: $$(cat /run/secrets/spejstore_clientid)
+      SPEJSTORE_SECRET: $$(cat /run/secrets/spejstore_secret)
+    volumes:
+      - .:/code
```

Comment:
That volume rather has no sense

Reply:
Yeah, need to figure out if it's even needed

Reply:
let's try and assume it isn't